variables.robot

```diff
@@ -221,6 +221,10 @@ ${OS_UBUNTU}=               ubuntu
 # NovaCustom V560TU -----------------------------------------------------------
 &{RTE62}=                   ip=0.0.0.0
 ...                         platform=novacustom-v560tu    platform_vendor=Clevo
+# Protectli V1211
+&{RTE63}=                   ip=192.168.10.227    cpuid=02c00042652f4120
+...                         platform=protectli-v1210
```

Comment:
it can include v1210, but will need 8G RAM instead of 4G

Reply:
Fixed in: 88bdb0d66fe6930762da1ec2d235e3fbf70506ef